File: 백수/코테공부/greedy/124나라의 숫자.py
#https://programmers.co.kr/learn/courses/30/lessons/12899
def solution(n):
    answer = ''
    li = '412'
    while n>0:
        tmp = n%3
        n = n//3
        answer = li[tmp]+answer
        if tmp == 0:
            n-=1
    return answer

File: 백수/algorithm/etc/124 나라의 숫자.java
class Solution {
    public String solution(int n) {
        String answer = "";
        String[] s = {"4","1","2"};
        while(n>0){
            int mod=n%3;
            n/=3;
            answer = s[mod]+answer;
            if(mod==0){
                n-=1;
            }
        }
        return answer;
    }
}